lib/controllers/DashboardController.js

```diff
@@ -28,6 +29,13 @@ class DashboardController extends BaseController {
         info = _.assign({
           userId: req.session.user_id,
         }, info);
+        let additional_info = {};
+        if (req.instance.plan.manager.settings.dashboard_template) {
+          additional_info = yaml.safeLoad(_.template(Buffer.from(req.instance.plan.manager.settings.dashboard_template, 'base64'))(info));
+        }
+        info = _.assign({
+          options: additional_info
```

Comment:
userId: req.session.user_id -- add this directly here.

Reply:
rename `options` to something more meaningful like `service_custom_attrs` or `custom_attrs`